bson/objectid.py

```diff
@@ -86,8 +66,7 @@ def __init__(self, oid=None):
         An ObjectId is a 12-byte unique identifier consisting of:
 
           - a 4-byte value representing the seconds since the Unix epoch,
-          - a 3-byte machine identifier,
-          - a 2-byte process id, and
+          - a 5-byte random number,
           - a 3-byte counter, starting with a random value.
 
         By default, ``ObjectId()`` creates a new unique identifier. The
```

Comment:
At the bottom can we add a .. versionchanged 3.8 that mentions we now implement ObjectId spec version 0.2? https://github.com/mongodb/specifications/blob/master/source/objectid.rst

And in the changelog too?